os-partition/added/partitionPV.sh

```diff
@@ -34,9 +34,24 @@ function partitionPV() {
             # Not terminating, grab the lock without waiting
             flock -n $LOCK_FD
           else
-            # Terminating, grab the lock with timeout
-            echo "Existing server instance is terminating, waiting to acquire the lock"
-            flock -w $LOCK_TIMEOUT $LOCK_FD
+            REMAINING_LOCKING_TIME=$LOCK_TIMEOUT
+            echo "Existing server instance is terminating, attempting lock per second for $LOCK_TIMEOUT seconds"
+
+            while : ; do
+              flock -n $LOCK_FD
+              if [$? -eq 0 ]; then
+                # Grabbed the lock successfully
+                break
+              fi
+              $REMAINING_LOCKING_TIME=$(expr $REMAINING_LOCKING_TIME - 1)
```

Comment:
I believe that we want to assign to `REMAINING_LOCKING_TIME`. $ shouldn't be there.

Reply:
@Crumby Correct, thanks, addressed below.